data_steward/cdr_cleaner/manual_cleaning_rules/remediate_basics.py

```diff
@@ -182,22 +195,48 @@
 SELECT
     i.person_id,
     i.src_id,
-    sb.state_of_residence_concept_id,
-    sb.state_of_residence_source_value,    
+    sb.state_of_residence_concept_id, -- This might be NULL if the person is new from the incremental dataset --
+    sb.state_of_residence_source_value, -- This might be NULL if the person is new from the incremental dataset -- 
     i.sex_at_birth_concept_id,
     i.sex_at_birth_source_concept_id,
     i.sex_at_birth_source_value
 FROM `{{project}}.{{incremental_dataset}}.{{table}}` i
-JOIN `{{project}}.{{sandbox_dataset}}.{{sandbox_table}}` sb
+LEFT JOIN `{{project}}.{{sandbox_dataset}}.{{sandbox_table}}` sb
 ON i.person_id = sb.person_id
+{% if exclude_lookup_dataset and exclude_lookup_table %}
+WHERE i.person_id NOT IN (
+    SELECT 
+        {% if is_deid %} research_id {% else %} person_id {% endif %}
+    FROM `{{project}}.{{exclude_lookup_dataset}}.{{exclude_lookup_table}}` 
+)
+{% endif %}
```

Comment:
@hiro-mishima and @ksdkalluri to make sure we are grabbing valid data you might try adding:
```python
AND i.person_id in (
    SELECT distinct person_id
    FROM `{{project}}.{{dataset}}.{{table}}
)
```